nncf/common/graph/patterns/patterns.py

```diff
@@ -80,6 +80,7 @@ class GraphPattern:
     NODE_TYPE_ATTR = "metatype"
     ANY_PATTERN_NODE_TYPE = "ANY_PATTERN_NODE"
     NON_PATTERN_NODE_TYPE = "NON_PATTERN_NODE"
+    NON_PATTERN_NODE_WITH_TYPE = "NON_PATTERN_NODE_WITH_TYPE"
```

Comment:
I am not sure that this name looks good. `NON_PATTERN_NODE_TYPE` is a special name which means that this node is not included in the pattern but has edges leading to that pattern. Your proposal attribute means that the node is actually inside the pattern but should be excluded after matching.
What do you think about `PATTERN_NODE_TO_EXCLUDE`?

Reply:
Done